server/src/test/java/org/elasticsearch/update/UpdateIT.java

```diff
@@ -293,12 +293,8 @@ public void testUpdate() throws Exception {
         ensureGreen();
 
         Script fieldIncScript = new Script(ScriptType.INLINE, UPDATE_SCRIPTS, FIELD_INC_SCRIPT, Collections.singletonMap("field", "field"));
-        try {
-            client().prepareUpdate(indexOrAlias(), "type1", "1").setScript(fieldIncScript).execute().actionGet();
-            fail();
-        } catch (DocumentMissingException e) {
-            // all is well
-        }
+        expectThrows(DocumentMissingException.class,
```

Comment:
:heart:

Reply:
Usually I do
```
Exception e = expectThrows(DocumentMissingException.class, () ->
    the thing);
assertEquals("whatever", e.getMessage());
```

Just to make sure it was the *right* exception. In this case there is only one kind of message for `DocumentMissingException` so it is probably safe to do what you are doing, but it still might be nice anyway.